config.mk

```diff
@@ -1,3 +1,11 @@
+# vim: tabstop=8 shiftwidth=8 noexpandtab:
+#
+# This is a common makefile used to establish the target architecture
+# and set the default value of the lock dependency validator flag.
+#
+# The following make variables are assumed to be set:
+# -BOARD: The target board.
```

Comment:
It's not assumed to be set, since the makefile sets the default value to `malta`.

Reply:
I've changed the description form `The following make variables are assumed to be set` to `The following make variables are set by the including makefile` and specified that the variable has a default value.